BlockchainSdk/WalletManagers/NEAR/NEARAddressService.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  NEARAddressService.swift
+//  BlockchainSdk
+//
+//  Created by Andrey Fedorov on 30.10.2023.
+//  Copyright © 2023 Tangem AG. All rights reserved.
+//
+
+import Foundation
+
+final class NEARAddressService {
+    private let blockchain: Blockchain
+    private lazy var walletCoreAddressService = WalletCoreAddressService(blockchain: blockchain)
+
+    init(blockchain: Blockchain) {
+        self.blockchain = blockchain
+    }
```

Comment:
Я за такой вариант, для других блокчейнов не актуально 

```suggestion
    private lazy var walletCoreAddressService = WalletCoreAddressService(blockchain: .near)

    init() {}
```

Reply:
Блокчейн это не просто енам, а еще содержит кривую и признак тестнета - поэтому мне не очень хочется передавать их отдельно тут и в других аналогичных местах по двум причинам:
1. Архитектурная - зачем юзать более специфик типа (`curve` и флаг `isTestnet`), когда можно юзать более абстрактные - `Blockchain`?
2. В будущем `Blockchain` может быть расширен еще каким-то образом, соответственно придется прокидывать сюда и в аналогичные места все больше параметров.

Reply:
а что будет если передать сюда .bitcoin?

Reply:
Я просто специально все AddressService переделывал что они работали только с одной сетью 
У нас же специально есть фабрика, где как раз идет выбор сервиса
В идеале может как то в будущем избавляться от `enum` и переходить на подобие namespaces для каждого блокчейна
Что бы по одному паттерну все было, как это у WalletCore сейчас

Reply:
Предлагаешь не юзать сonvenience инит в WalletCoreAddressService и передавать тип сети явно?

Reply:
ага, это же `NEARAddressService.swift` пускай в нем будет только `Near` )

Reply:
Поправил